src/python/pants/engine/scheduler.py

```diff
@@ -406,15 +406,25 @@ def visualize_graph_to_file(self, filename):
   def visualize_rule_graph_to_file(self, filename):
     self._scheduler.visualize_rule_graph_to_file(filename)
 
+  def execution_request_literal(self, request_specs):
+    native_execution_request = self._scheduler._native.new_execution_request()
+    for subject, product in request_specs:
+      self._scheduler.add_root_selection(native_execution_request, subject, product)
+    return ExecutionRequest(request_specs, native_execution_request)
+
+  # FIXME: note that this does a cross prod!
```

Comment:
You updated the comment here... can remove.

Reply:
Done in 5a1665b!